lucene/analysis/kuromoji/src/java/org/apache/lucene/analysis/ja/JapaneseNumberFilter.java

```diff
@@ -209,6 +216,9 @@ public final boolean incrementToken() throws IOException {
       if (moreTokens) {
         term = termAttr.toString();
         numeralTerm = isNumeral(term) || isNumeralPunctuation(term);
+        if (isNumeral(term)) {
+          // lastNumeralTokenState = captureState();
+        }
```

Comment:
What does this block do?

Reply:
Sorry, this shouldn't be left out, it's a left over from when I was doing some local debugging.
I'll push an update shortly. The classes javadocs say that the filter "will inherit the values of the last token used to compose the normalized number", and in order to do so, that state is saved here on every new numeral token encountered.